docs/DeveloperGuide.adoc

```diff
@@ -48,12 +66,17 @@ This will generate all resources required by the application and tests.
 
 === Verifying the setup
 
+To confirm that you've successfully set up the project, do take the time to ensure
+that your set up is correct and ready to run.
```

Comment:
Same as above

Reply:
We should have a more concrete example to verify correct setup. 
Maybe a checklist that developers have to go through to ensure that it is correct?